recipe/meta.yaml

```diff
@@ -102,9 +102,8 @@ requirements:
 # Anaconda has observed it on Windows, macOS, and Linux.
 {% set tests_to_skip = tests_to_skip + " or test_apply_parallel_rgb" %}
 
-{% set tests_to_skip = tests_to_skip + " or test_thresholding" %}
+{% set tests_to_skip = tests_to_skip + " or test_thresholding or test_uint8 or test_float" %}  # [win64]
```

Comment:
Why we now run test_thresholding on all platforms?  I

Reply:
but on win64?

Reply:
Or are those test just needed to be skipped on win64?

Reply:
Are those test passing on win32?

Reply:
Yeah, I skip them only on win64. But new errors with new tests appear again so I skip them to find all tests that cause errors.
I can't build scikit-image because of `tifffile ` on `win32` has a version less than it should be